package-lock.json

```diff
@@ -0,0 +1,10853 @@
+{
```

Comment:
Do we want to include a `lock` file in the repo?

Reply:
It is required to speed up CI via [`npm ci`](https://docs.npmjs.com/cli/ci.html)